src/dev/build/tasks/copy_source_task.ts

```diff
@@ -30,6 +30,11 @@ export const CopySource: Task = {
         '!src/functional_test_runner/**',
         '!src/dev/**',
         '!**/jest.config.js',
+        '**/jest.integration.config.js',
```

Comment:
Should this have a !?

Reply:
Thanks - fixed with https://github.com/elastic/kibana/pull/127901/commits/3f8f583a689df984e043fb0b0894124cde817e3d